### DISCUSSION TITLE
CUDA11.4 安装报错,提示404

### DISCUSSION THREAD

**Discussion by MdcGIt** [InternLM/lmdeploy#2176] [General]
CUDA11.4 安装报错,提示404
![image](https://github.com/user-attachments/assets/ce3144a6-b878-462d-90e9-5ad0227eb702)

**Comment by lvhan028** [comment]
在 issue https://github.com/InternLM/lmdeploy/issues/2175 中回复了。可以重新试试看了

**Reply by MdcGIt** [reply]
好的

**Reply by MdcGIt** [reply]
![image](https://github.com/user-attachments/assets/138a2cf2-cf4f-472c-af3f-da8108ad88e0)
这我是不是要更新显卡驱动了。。

**Reply by lvhan028** [reply]
Can you share the information by executing `nvidia-smi`?

**Reply by MdcGIt** [reply]
![image](https://github.com/user-attachments/assets/be2d3649-c339-48a1-8386-0bb4890ac03e)

**Comment by lvhan028** [comment]
从上面的日志看，是pytorch 判断 driver 版本太低了。
你可以尝试降低 pytorch 版本。https://pytorch.org/get-started/previous-versions/
```shell
pip install torch==1.12.0+cu113 torchvision==0.13.0+cu113 --extra-index-url https://download.pytorch.org/whl/cu113
```

**Reply by MdcGIt** [reply]
好的 我试下

**Reply by MdcGIt** [reply]
![image](https://github.com/user-attachments/assets/32195255-16e8-4a42-bdef-aa9cfe1e5888)
昨天还能安装,今天我在一个新的conda环境上就不行了

**Reply by lvhan028** [reply]
python --version 看下是什么版本

**Reply by MdcGIt** [reply]
![image](https://github.com/user-attachments/assets/9d377e7b-4e10-4f6e-9949-ef053e90f67c)

**Reply by lvhan028** [reply]
那你应该要 export PYTHON_VERSION=310

**Reply by MdcGIt** [reply]
好的 已经安装成功了 我怎么在启动的时候限制上下文的最大长度呢

**Reply by lvhan028** [reply]
希望这份文档能帮到你 https://lmdeploy--2038.org.readthedocs.build/en/2038/get_started.html#llm-inference

**Reply by MdcGIt** [reply]
![image](https://github.com/user-attachments/assets/493890de-a3ff-4406-a0ba-08496180183b)
我有俩张显卡,一张T4 一张rtx3060 为什么T4的卡不工作呢,运行了一个qwen的小参数模型

**Reply by lvhan028** [reply]
执行命令是什么呢？

**Reply by MdcGIt** [reply]
CUDA_VISIBLE_DEVICES=0,1  lmdeploy chat  /sdd/model/lmdeploy/model/Qwen2-1.5B  --cache-max-entry-count 0.1 --tp=2 
今天执行报错了: 
![image](https://github.com/user-attachments/assets/dd9fb1e0-4484-410d-b6eb-34d7467d5c1f)

**Reply by MdcGIt** [reply]
--tp=2 这个参数加了就报错了 不加能正常回答 但是只有RTX3060ti这张显卡在用  T4那张就没用了

**Reply by lvhan028** [reply]
@lzhangzz do you have any idea? One T4, one 3060, different arch, emmmm

**Reply by lzhangzz** [reply]
The model is in BF16 format which is not supported by the T4 GPU.

**Reply by MdcGIt** [reply]
我跑glm4-9b-chat也是  俩张卡跑不起来  只有GPU1 在使用  
![image](https://github.com/user-attachments/assets/1abae79c-8aae-4c9e-b8d8-f572e9228099)
![image](https://github.com/user-attachments/assets/b60649da-eeac-45ff-a195-02a8339cd1da)
